response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -101,7 +106,11 @@ def create_collection_exercise_event(collection_exercise_id, tag, timestamp):
         if response.status_code == 400:
             logger.warning('Bad request creating event', collection_exercise_id=collection_exercise_id,
                            tag=tag, timestamp=formatted_timestamp, status=response.status_code)
-            return False
+            response_content = response.content.decode("utf-8")
+            response_json = json.loads(response_content)
+            logger.error('got error', message=response_json['error']['message'])
```

Comment:
Should say something else besides `got error`

Reply:
I feel like this should be rolled into the logger.warning message above. And the warning should be an error? I don't know the context of this well enough to say that it should be an error, but it feels like one.